sdk/communication/Azure.Communication.CallingServer/src/Azure.Communication.CallingServer.csproj

```diff
@@ -30,7 +30,7 @@
     <Compile Include="$(AzureCoreSharedSources)DiagnosticScopeFactory.cs" LinkBase="Shared" />
     <Compile Include="$(AzureCoreSharedSources)HttpMessageSanitizer.cs" LinkBase="Shared" />
     <Compile Include="$(AzureCoreSharedSources)RetriableStream.cs" LinkBase="Shared" />
-    <Compile Include="$(AzureCoreSharedSources)CancellationHelper.cs" LinkBase="SharedCore" />
+    <Compile Include="$(AzureCoreSharedSources)CancellationHelper.cs" LinkBase="Shared" />
```

Comment:
I'm just curious why this didn't fail going in

Reply:
It is not a functional issue. It just puts the file in a different folder.